flexget/components/sites/sites/lostfilm.py

```diff
@@ -206,7 +206,10 @@ def on_task_input(self, task, config):
             try:
                 response = task.requests.get(redirect_url, params=params)
             except RequestException as e:
-                logger.error('Failed to get lostfilm download page: {:s}'.format(e))
+                logger.error('Failed to get the redirect page: {}', e)
+                continue
+            except Exception as e:
```

Comment:
Generally it's not good to handle unknown exceptions. Is there something specific this was trying to catch? Adding the general handler will cover up exceptions from the log which should be handled explicitly, or in some other way, and make debugging harder later.

Reply:
cfscraper produce an exception if enabled.
I'm trying to avoid conditionally import it.

Reply:
Aha. Well at a minimum, I'd add a comment explaining that.

Reply:
Fixed